### ISSUE TITLE
Implement Mockingboard

### ISSUE THREAD

**Issue by bradgrantham** [bradgrantham/apple2e#29]
Get @bartgrantham to write a cc65-based YM player to test the emulator.

**Comment by bartgrantham** [comment]
Does the emulator emulate a Mockingboard?  ;)

**Comment by bradgrantham** [comment]
Not yet.  :-)

**Comment by bartgrantham** [comment]
Are you waiting on a working YM player that is known to work on real hardware?

**Comment by bradgrantham** [comment]
No.  Just noting that I should get you to write the YM player once the
emulator emulates a Mockingboard.

--
Brad Grantham, http://plunk.org/~grantham/
